magefile.go

```diff
@@ -342,13 +352,18 @@ func patchPackageOperatorManager(scheme *k8sruntime.Scheme, obj *unstructured.Un
 		panic(fmt.Errorf("converting to Deployment: %w", err))
 	}
 
-	var packageOperatorManagerImage string
+	var (
+		packageOperatorManagerImage string
+		remotePhasePackageImage     string
+	)
 	if len(os.Getenv("USE_DIGESTS")) > 0 {
 		// To use digests the image needs to be pushed to a registry first.
-		mg.Deps(mg.F(Build.PushImage, "package-operator-manager"))
+		mg.Deps(mg.F(Build.PushImage, "package-operator-manager", remotePhasePackageName))
 		packageOperatorManagerImage = locations.ImageURL("package-operator-manager", true)
+		remotePhasePackageImage = locations.ImageURL(remotePhasePackageName, true)
 	} else {
 		packageOperatorManagerImage = locations.ImageURL("package-operator-manager", false)
```

Comment:
Should running `Dev.LoadImage` on the images be a dependency for the case where we don't use digests?

Reply:
I want all of this to be refactored and either moved into devkube or the CLI.
Especially the digest handling needs to be moved into the CLI, because I am not sure it works as intended and is very brittle...

I don't think loading images in here is necessary.